src/components/Keyboard/Keyboard.jsx

```diff
@@ -0,0 +1,4380 @@
+import React from 'react'
+import styles from "./Keyboard.module.css";
+
+export default function Keyboard() {
+
+    function KeyText({ fill, symbolX, symbolY, children }) {
+        return (
+            <text
+                fill={ fill }
+                xmlSpace="preserve"
+                style={{ whiteSpace: "pre" }}
+                fontFamily="Open Sans"
+                fontSize="11"
+                fontWeight="600"
+                letterSpacing="0em"
+            >
+                <tspan x={symbolX} y={symbolY}>
+                    {children}
+                </tspan>
+            </text>
+        )
+    }
+
+    function Key ({ base, alt, shift, altShift, keyX, keyY }) {
+        return (
+            <g id="Key">
+                <rect
+                    width="41"
+                    height="41"
+                    x={keyX}
+                    y={keyY}
+                    fill="#fff"
+                    stroke="#646464"
+                    rx="2.5"
+                ></rect>
+                <KeyText fill="#671584" x="36.734" y="43.267">{base}</KeyText>
+                <KeyText fill="#671584" x="36.734" y="63.267">{alt}</KeyText>
+                <KeyText fill="#671584" x="16.734" y="43.267">{shift}</KeyText>
+                <KeyText fill="#671584" x="16.734" y="63.267">{altShift}</KeyText>
+            </g>
+        )
+    }
+    const keys = "qwertyuiop[]";
+    const alt = "qwertyuiop[]";
+    const shift = "QWERTYUIOP{}";
+    const altShift = "QWERTYUIOP{}";
+
+return (
+    <>
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="718"
+            height="274"
+            fill="none"
+            viewBox="0 0 718 274"
+        >
+            <path fill="#E5E5E5" d="M0 0H718V274H0z"></path>
+            <g clipPath="url(#clip0_358_2616)">
+                <path
+                    fill="#fff"
+                    d="M0 0H1233V1248H0z"
+                    transform="translate(-61 -243)"
+                ></path>
+                <g>
+                    <rect
+                        width="716"
+                        height="272"
+                        x="1"
+                        y="1"
+                        stroke="#777"
+                        rx="8"
+                    ></rect>
+                        <g>
+                            <g>
+                                <Key key={keys[0]} alt={alt[0]} shift={shift[0]} altShift={altShift[0]} keyX="9.5" keyY="28.5" />
```

Comment:
Actually you can count `x` and `y` dynamically like

x = offsetX + index * (gapX + width)
y = offsetY + index * (gapX + height)